tools/collect_pylint_input_files_for_backend.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
Use `pathlib` and `Path` methods everywhere - you won't need to use `os.sep` and `os.basename` then, and the path handling code will be better and prettier.

Reply:
Done, please check